src/tree-node/index.js

```diff
@@ -29,7 +29,8 @@ const getNodeCx = props => {
       'match-in-children': keepTreeOnSearch && matchInChildren,
       'match-in-parent': keepTreeOnSearch && keepChildrenOnSearch && matchInParent,
       partial: showPartiallySelected && partial,
-      readOnly
+      readOnly,
+      checked
```

Comment:
Umm, why do we need this? As a courtesy?

Reply:
If you want to override the rendering of checked nodes. It's now currently difficult to do so by css since a selector which checks if a child is checked is not possible.